lib/init.js

```diff
@@ -84,6 +87,11 @@ module.exports = {
           }
         });
 
+        // Return if document has changed
```

Comment:
A `null` return is a special value that tells Linter not to update it's cache of messages, it's not required to note that though 😛.